packages/fixtures/src/events.ts

```diff
@@ -81,3 +86,23 @@ export const appMention = (
     ...options,
   });
 };
+
+export const memberJoinedChannel = (
+  user: string,
+  channel: string,
+  channel_type: string,
+  team: string,
+  options: Partial<MemberJoinedChannelEvent> = {},
+): EnvelopedEvent<MemberJoinedChannelEvent> => {
+  const { inviter } = fields;
+
+  return apiEvent<MemberJoinedChannelEvent>({
+    type: 'member_joined_channel',
+    user,
+    channel,
```

Comment:
since this is fixtures for testing purposes, I think we could potentially simplify this function for users and pull `user`, `team`, and `channel` ids from fields. They're all overrideable via opts anyway. That way you'd be able to get an example `memberJoinedChannel` event just by calling e.g.

`events.memberJoinedChannel()`.

I'm thinking we can then just automatically set `channel_type` based on any options they provide, but still override. Something like

```ts
export const memberJoinedChannel = (
  options: Partial<MemberJoinedChannelEvent> = {},
): EnvelopedEvent<MemberJoinedChannelEvent> => {
  const { channel: {id: channel},  inviter: {id: inviter}, team: {id: team}, user: {id: user} } = fields;
 
  const channel_type = options.channel? options.channel.charAt(0) : channel.charAt(0);

  return apiEvent<MemberJoinedChannelEvent>({
    type: 'member_joined_channel',
    channel,
    channel_type,
    inviter,
    team,
    ...options,
  });
};
```
This would allow for all of: `events.memberJoinedChannel()`, `events.memberJoinedChannel({channel: 'G123456'})`, and `events.memberJoinedChannel({channel: 'G123456', 'channel_type': 'C'})`.

Reply:
Thanks 👍